### ISSUE TITLE
GoLang Binary Issue

### ISSUE THREAD

**Issue by def3n** [mitre/sandcat#359]
When running a the Sandcat GoLang agent for linux, it fails. Looks like the Go binary breaks when it is decoding the http response (probably over ssl cert).  Is there a fix available for this issue? 

```
Starting sandcat in verbose mode.
[*] Attempting to set channel HTTP
Beacon API=/beacon
[*] Set communication channel to HTTP
initial delay=0
server=https://xxxxxxxxxx/file/download
group=red
privilege=User
allow local p2p receivers=false
beacon channel=HTTP
[-] Failed to decode HTTP response: illegal base64 data at input byte 3
[-] beacon: DEAD
[-] Failed to decode HTTP response: illegal base64 data at input byte 3
[-] beacon: DEAD
[-] Failed to decode HTTP response: illegal base64 data at input byte 3
[-] beacon: DEAD
```

**Comment by garunagiri** [comment]
Hi def3n,

I have a couple questions that could help us figure out this issue.

What release of CALDERA are you running?
Do you have GoLang installed on your server?
If so, was the instance of the agent dynamically compiled, or was it the version that is shipped with the CALDERA code?

**Comment by def3n** [comment]
Thanks for the response. 

- Caldera version is 3.0.0 
- GoLang is installed on the server. Version 1.13.9 (linux/amd64)
- Used the agent that was shipped with the Caldera code

**Comment by def3n** [comment]
Is there any documentation which I can refer to for this?

**Comment by garunagiri** [comment]
At the moment there is no documentation regarding the usage of the `-l DEBUG` flag unfortunately.

**Comment by b17zr** [comment]
We have some troubleshooting documentation coming in the next release, in case that helps: https://github.com/mitre/fieldmanual/blob/master/sphinx-docs/Troubleshooting.md

**Comment by fDotGoodman** [comment]
Hi - Adding for visibility for future viewers of this issue, should anyone else encounter it. 
**Tl;Dr - I encountered this issue when trying to install the server, having deleted the files Stockpile, even though I disabled it in the server config**

I encountered this issue when trying to install Caldera on Ubuntu 22.04. During install, there were a number of plugins that I didn't want to be included. My incorrect assumption here was that after removing the lines in the server config file that loaded them, I could safely remove those plugins' directories themselves. There is nothing in the documentation that says this is, in reality the doco states clearly during the install that you should clone the repo recursively, and disable the plugins you don't want to run, it does not state that it's safe to delete the plugin directories that aren't loaded.

In case anyone else encounters this issue and has also removed the Stockpile plugin directory, do note that there appear to be dependencies between at least this and the main server, even if it isn't enabled.
